src/py/flwr/client/app.py

```diff
@@ -667,3 +671,22 @@ def signal_handler(sig, frame):  # type: ignore
 
         signal.signal(signal.SIGINT, signal_handler)
         signal.signal(signal.SIGTERM, signal_handler)
+
+
+def run_clientappio_api_grpc(
+    address: str = "0.0.0.0:9094",
```

Comment:
We usually put the default in a constant, sth like `DEFAULT_ADDRESS_CLIENTAPPIO = ...`

Reply:
I introduced it in the next PR. Let me shift it here.